src/features/web-compat.js

```diff
@@ -5,6 +5,11 @@ import { URL } from '../captured-globals.js'
  * Fixes incorrect sizing value for outerHeight and outerWidth
  */
 function windowSizingFix () {
+    // macOS browser incorrectly reports window height for window.screenY / window.screenTop
+    if (window.screenY === window.screen.height) {
+        window.screenY = window.screenTop = 0
```

Comment:
`0` isn't a valid value here, it should be the position of the content within the screen. So for me Safari is at it's minimum `38` due to the browser chrome being in the way.

I would suggest we either make this value remotely configurable to adjust the geometry of all of these values.

`outerHeight` is `0` despite it being a size that should always be bigger than innerHeight. This value should actually be the same offset when the browser is fullscreen as `screenTop`/`Y`

Reply:
As mentioned in the Asana task, our extension sets `screenY/screenTop` to `0`, just verified it again in Chrome and Firefox.  

`0` is actually a valid value here, when you are in full-screen mode.  If we want to make it configurable we can but it can vary per system anyway (`screenY` is `24` for me in Chrome ATM), so `0` seemed like a reasonable default.

Reply:
Re `outerHeight` are you saying we should change this existing behavior?  I can update that if so.

Reply:
Updated to use the `windowSizing.screenYOverride` setting if defined, or `screen.availTop` if not.